### PR TITLE
DOC: Numpydoc format.

### PR DESCRIPTION
**Pull request by Carreau**
Returns is supposed to be a description list to be properly parsed by
numpydoc, otherwise each line is parsed as an individual item.

- [x] Documentation is sphinx and numpydoc compliant

### CODE REVIEW

File: lib/matplotlib/__init__.py [hunk 1]
```diff
@@ -273,9 +273,11 @@ def _get_executable_info(name):
 
     Returns
     -------
-    If the executable is found, a namedtuple with fields ``executable`` (`str`)
-    and ``version`` (`distutils.version.LooseVersion`, or ``None`` if the
-    version cannot be determined).
+
+    tuple or None:
```
Comment (timhoffm) [inline lib/matplotlib/__init__.py:277]:
```suggestion
    tuple or None
```

File: lib/matplotlib/mlab.py [hunk 2]
```diff
@@ -934,10 +934,11 @@ def cohere(x, y, NFFT=256, Fs=2, detrend=detrend_none, window=window_hanning,
 
     Returns
     -------
-    The return value is the tuple (*Cxy*, *f*), where *f* are the
-    frequencies of the coherence vector. For cohere, scaling the
-    individual densities by the sampling frequency has no effect,
-    since the factors cancel out.
+
+    tuple (*Cxy*, *f*)
```
Comment (timhoffm) [inline lib/matplotlib/mlab.py:938]: Synchronize return docs with https://matplotlib.org/devdocs/api/_as_gen/matplotlib.axes.Axes.cohere.html

File: lib/matplotlib/__init__.py (hunk 1 above)
Comment (Carreau) [reply lib/matplotlib/__init__.py:277]: From numpydoc:
`[...]The colon must be preceded by a space, or omitted if the type is absent.` Which, usually is when you have a name for the value and the example n numpydoc:

```
Returns
-------
int
    Description of anonymous integer return value.
```

So I think that no, there is no `:`. I'm happy to put it.

Comment (Carreau) [reply lib/matplotlib/__init__.py:277]: Sorry I misread the diff. you are right. I'm agreeing with you.

File: lib/matplotlib/mlab.py (hunk 2 above)
Comment (Carreau) [reply lib/matplotlib/mlab.py]: done.

File: lib/matplotlib/__init__.py [hunk 3]
```diff
@@ -273,9 +273,11 @@ def _get_executable_info(name):
 
     Returns
     -------
-    If the executable is found, a namedtuple with fields ``executable`` (`str`)
-    and ``version`` (`distutils.version.LooseVersion`, or ``None`` if the
-    version cannot be determined).
+
+    tuple or None
+        If the executable is found, a namedtuple with fields ``executable``
```
Comment (timhoffm) [inline lib/matplotlib/__init__.py:278]:
```suggestion
    tuple
        A namedtuple with fields ``executable``
```
This always returns `_ExecInfo` (or raises).

Please also re-wrap.

**Pull request merged**